### PR TITLE
Update SetMpGamerTagEnabled.md

### PR DESCRIPTION
**Pull request by TomStilson**
Corrected the native name and make it more precise.

### CODE REVIEW

**Review by jaymo1011 — CHANGES_REQUESTED** [review]
Thanks for documenting! :)

File: HUD/SetMpGamerTagEnabled.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 ```
 
 ## Parameters
-* **gamerTagId**: 
-* **toggle**: 
+* **gamerTagId**: Gamer tag handle.
+* **toggle**: Set to ```true``` to enable all elements for the gamer tag, ```false``` to disable all elements.
 
+
+Toggles all elements for the gamer tag provided.
```
[the remark this answers is not in the document]
Comment (TomStilson) [reply HUD/SetMpGamerTagEnabled.md:17]: Got it, thanks!

File: HUD/SetMpGamerTagEnabled.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 // 0xEE76FF7E6A0166B0 0x767DED29
-void _SET_MP_GAMER_TAG_ENABLED(int gamerTagId, BOOL toggle);
+void _SET_MP_GAMER_TAG_ENABLE_ALL_ELEMENTS(int gamerTagId, BOOL toggle);
 ```
 
 ## Parameters
-* **gamerTagId**: 
-* **toggle**: 
+* **gamerTagId**: Gamer tag handle.
+* **toggle**: Set to ```true``` to enable all elements for the gamer tag, ```false``` to disable all elements.
```
[the remark this answers is not in the document]
Comment (TomStilson) [reply HUD/SetMpGamerTagEnabled.md:14]: Got it, thank you!

**Review by stannum-cfx — CHANGES_REQUESTED** [review]
Merge conflicts should be fixed before merging.

Also, since this native has a known Joaat hash, someone might be able to brute force the actual name so we don't have to keep adding more aliases? This isn't mandatory, but would be nice.

**Pull request closed without merging**